src/arch/mips/kernel/entry.S

```diff
@@ -21,13 +21,24 @@
  */
  LEAF(mips_second_exception_handler)
 	SAVE_ALL                             /* save all needed registers */
-
+	li $t1, 0x40000000
```

Comment:
Add some comment please to explain the code

Reply:
done